debian-s390x/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM s390x/debian:9
+LABEL MAINTAINER="OSU Open Source Lab <dockerhub@osuosl.org>"
+
+# Below was taken from https://github.com/jenkinsci/docker-ssh-slave
+
+ARG user=jenkins
+ARG group=jenkins
+ARG uid=1000
+ARG gid=1000
+ARG JENKINS_AGENT_HOME=/home/${user}
+
+ENV JENKINS_AGENT_HOME ${JENKINS_AGENT_HOME}
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+ENV PATH $PATH:$JAVA_HOME/bin
+
+RUN groupadd -g ${gid} ${group} \
```

Comment:
It looks like `groupadd` might not be available.

```
Step 11/28 : RUN groupadd -g ${gid} ${group}     && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
 ---> Running in c7f920361706
standard_init_linux.go:185: exec user process caused "no such file or directory"
The command '/bin/sh -c groupadd -g ${gid} ${group}     && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"' returned a non-zero code: 1
```

Reply:
Are you trying to run this locally? If so this won't work since this needs to run on IBM-Z.